migrations/011.do.top-measurements-stations-mv.sql

```diff
@@ -0,0 +1,10 @@
+CREATE MATERIALIZED VIEW top_measurement_stations_mv AS
```

Comment:
Can we please find a name that conveys that these stats are for one day only?

The rows are keyed by `participant_address`, so perhaps we should use "participants" and not "stations" in the name.